addons/ofxAndroid/ofAndroidLib/src/cc/openframeworks/OFAndroidWindow.java

```diff
@@ -249,13 +337,13 @@ public OFGLSurfaceView(Context context,AttributeSet attributes) {
 //        The Android SurfaceView source code is a bit confusing  - there are many times when some kind of surface is beign destroyed (eg. during window resize),
 //            so it is not that surprising, that not every surfaceDestoyed callback means what we might think it means.
 //        We don't need this callback that much anyways, the renderer does not call render callbacks when gl context is invalid, so the OFAndroidWindow.onSurfaceCreated callback should be enought for us to make things work.
-/*
+
     @Override
 	public void surfaceDestroyed(SurfaceHolder holder) {
     	super.surfaceDestroyed(holder);
 		OFAndroid.onSurfaceDestroyed();
 	}
-*/
+
```

Comment:
Just wanted to check it was intentional to uncomment surfaceDestroyed callback?
Considering the note above.  

If it is needed, then this PR looks good to go to me.

Reply:
Actually this is another Fix where Shader's are killed and never re-loaded with this commented out